backend/src/database/seeds/04_contacts.js

```diff
@@ -0,0 +1,50 @@
+exports.seed = function (knex) {
+  // Deletes ALL existing entries
+  return knex("contacts")
+    .del()
+    .then(function () {
+      // Inserts seed entries
+      return knex("contacts").insert([
```

Comment:
if you make this array a const at the top, we could potentially export it, and use it in tests for test data.